packages/workbox-background-sync/test/browser/background-sync-queue-plugin.js

```diff
@@ -3,25 +3,38 @@
  Licensed under the Apache License, Version 2.0 (the "License");
  you may not use this file except in compliance with the License.
  You may obtain a copy of the License at
- http://www.apache.org/licenses/LICENSE-2.0
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
  Unless required by applicable law or agreed to in writing, software
  distributed under the License is distributed on an "AS IS" BASIS,
  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  See the License for the specific language governing permissions and
  limitations under the License.
- */
+*/
 
 /* eslint-env mocha, browser */
-/* global chai, workbox */
+/* global chai */
```

Comment:
Ahh, it looks like @prateekbh is using `chai.assert` and everywhere else we use `chai.expect`, which is aliased to just `expect`.

I can remove this in all packages expect background sync, but we should probably be consistent everywhere.